mergify_engine/rules/__init__.py

```diff
@@ -411,6 +412,7 @@ def get_annotations(self, path: str) -> typing.List[github_types.GitHubAnnotatio
         return []
 
 
+@tracer.wrap("stream_yaml_load", span_type="worker")
```

Comment:
```suggestion
@tracer.wrap("yaml.load")
```
1. why stream?
2. don't set a type here; worker is not great but I don't think any other is great either.